falco/templates/pod-template.tpl

```diff
@@ -0,0 +1,326 @@
+  labels:
+    {{- include "falco.selectorLabels" . | nindent 4 }}
+    {{- with .Values.podLabels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+    checksum/rules: {{ include (print $.Template.BasePath "/rules-configmap.yaml") . | sha256sum }}
+    {{- if and .Values.certs (not .Values.certs.existingSecret) }}
+    checksum/certs: {{ include (print $.Template.BasePath "/certs-secret.yaml") . | sha256sum }}
+    {{- end }}
+    {{- with .Values.podAnnotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  serviceAccountName: {{ include "falco.serviceAccountName" . }}
+  {{- with .Values.podSecurityContext }}
+  securityContext:
+    {{- toYaml . | nindent 4}}
+  {{- end }}
+  {{- if .Values.driver.enabled }}
+  {{- if and (eq .Values.driver.kind "ebpf") .Values.driver.ebpf.hostNetwork }}
+  hostNetwork: true
+  dnsPolicy: ClusterFirstWithHostNet
+  {{- end }}
+  {{- end }}
+  {{- if .Values.priorityClassName }}
+  priorityClassName: {{ .Values.priorityClassName }}
+  {{- end }}
+  {{- with .Values.nodeSelector }}
+  nodeSelector:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.affinity }}
+  affinity:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.tolerations }}
+  tolerations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.imagePullSecrets }}
+  imagePullSecrets: 
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  containers:
+    - name: {{ .Chart.Name }}
+      image: {{ include "falco.image" . }}
+      imagePullPolicy: {{ .Values.image.pullPolicy }}
+      resources:
+        {{- toYaml .Values.resources | nindent 8 }}
+      securityContext:
+        {{- include "falco.securityContext" . | nindent 8 }}
+      args:
+        - /usr/bin/falco
+        {{- with .Values.collectors }}
+        {{- if .enabled }}
+        {{- if .containerd.enabled }}
+        - --cri
+        - /run/containerd/containerd.sock
+        {{- end }}
+        {{- if .crio.enabled }}
+        - --cri
+        - /run/crio/crio.sock
+        {{- end }}
+        {{- if .kubernetes.enabled }}
+        - -K
+        - {{ .kubernetes.apiAuth }}
+        - -k
+        - {{ .kubernetes.apiUrl }}
+        {{- if .kubernetes.enableNodeFilter }}
+        - --k8s-node
+        - "$(FALCO_K8S_NODE_NAME)"
+        {{- end }}
+        {{- end }}
+        - -pk
+        {{- end }}
+        {{- end }}
+    {{- with .Values.extra.args }}
+      {{- toYaml . | nindent 8 }}
+    {{- end }}
+      env:
+        - name: FALCO_K8S_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+      {{- if or (not .Values.driver.enabled) (and .Values.driver.loader.enabled .Values.driver.loader.initContainer.enabled) }}
+        - name: SKIP_DRIVER_LOADER
+          value:
+      {{- end }}
+      {{- if and .Values.driver.enabled (eq .Values.driver.kind "ebpf") }}
```

Comment:
```suggestion
      {{- if and .Values.driver.enabled .Values.driver.ebpf.path (eq .Values.driver.kind "ebpf") }}
```

Reply:
In order to set `FALCO_BPF_PROBE` env variable we also need the PATH to the `bpf` probe.